app/(auth)/landing.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+	StyleSheet,
+	Text,
+	TouchableOpacity,
+	Animated,
+	SafeAreaView,
+	View,
+} from "react-native";
+import React, { Dispatch, SetStateAction, useRef, useEffect } from "react";
+
+// props type
+interface LandingProps {
+	setShowTabs: Dispatch<SetStateAction<boolean>>;
+	headerText: string;
+	actionText: string;
+	imageURL: any;
+	onPressAction: () => void;
+}
+
+// Landing component
+const Landing: React.FC<LandingProps> = ({
+	headerText,
+	actionText,
+	onPressAction,
+	imageURL,
+}) => {
+	const fadeAnim = useRef(new Animated.Value(0)).current;
+
+    // animation effect
+	useEffect(() => {
+		Animated.timing(fadeAnim, {
+			toValue: 1,
+			duration: 1000,
+			useNativeDriver: true,
+		}).start();
+	}, [fadeAnim]);
+
+	return (
+		<SafeAreaView style={styles.container}>
+			<Text style={styles.headerText}>{headerText}</Text>
+            {/* rendering image or placeholder based on props */}
+			{imageURL !== "" ? (
+				<Animated.Image
+                    source={imageURL}
+					style={[styles.img, { opacity: fadeAnim }]}
+				/>
+			) : (
+				<View style={styles.placeholder}></View>
```

Comment:
Would rather you do this null check higher up, when unpacking the props. Minor but for future reference, avoid using long ternary operators in your JSX. Prefer ternaries before the return statement rather than inside of it.